IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -108,6 +108,10 @@ -(void)dealloc
     {
         [self.webView setNavigationDelegate:nil];
     }
+    if ([self.webView.UIDelegate isEqual:self])
```

Comment:
Is UIDelegate a property from Apple or from us?

Reply:
Its from Apple: https://developer.apple.com/documentation/webkit/wkuidelegate?language=objc

It allows us to have control over the browser-level camera prompting behavior later in this file